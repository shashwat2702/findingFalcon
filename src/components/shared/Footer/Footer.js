import React from 'react';
import './Footer.scss';

const Footer = () => (
  <div className="footer">
    <p>Footer</p>
  </div>
);
export default Footer;
