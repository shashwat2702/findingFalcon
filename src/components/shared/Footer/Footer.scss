.footer{
    position: fixed;
   left: 0;
   bottom: 0;
   width: 100%;
   background-color: #1ab188;;
   color: white;
   text-align: center;
}