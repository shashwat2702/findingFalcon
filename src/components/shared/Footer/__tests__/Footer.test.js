import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../Footer';

jest.mock('react-router-dom');

describe('Footer snapshot', () => {
  it('should render correctly', () => {
    const FooterSnap = renderer.create(<Footer />);
    const FooterJson = FooterSnap.toJSON();
    expect(FooterJson).toMatchSnapshot();
  });
});
