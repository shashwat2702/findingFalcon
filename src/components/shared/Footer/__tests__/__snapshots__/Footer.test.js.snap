// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot should render correctly 1`] = `
<div
  className="footer"
>
  <p>
    Footer
  </p>
</div>
`;
